backend/src/main/java/com/htecgroup/skynest/model/entity/InvalidJwtEntity.java

```diff
@@ -0,0 +1,17 @@
+package com.htecgroup.skynest.model.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.springframework.data.cassandra.core.mapping.PrimaryKey;
+import org.springframework.data.cassandra.core.mapping.Table;
+
+@Table
+@Getter
+@Setter
```

Comment:
Why do you need the setter?

Reply:
I thought I would, but I don't. I'll remove it.